.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Node CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Use Node.js latest
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: "*"
+      - name: Generate coverage report
+        run: |
+          make -j test-ci-coverage
+      - name: Upload coverage report
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+  legacy-node-version-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
```

Comment:
Did you mean `false`? `true` is default

Reply:
Oh I meant for failing fast. Didn't know this is default. Thanks.

Reply:
Why failing fast?

Reply:
They are strategy for legacy node version tests. I think they are highly correlated, if one of them is failed the other one will probably fail, too(Regardless of network issues or the change indeed will fail on node 8).